# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kphone/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur kphone-orig/kphone/kcallwidget.h kphone/kphone/kcallwidget.h
--- kphone-orig/kphone/kcallwidget.h	2005-04-18 13:04:40.000000000 +0000
+++ kphone/kphone/kcallwidget.h	2006-04-03 11:59:33.717210000 +0000
@@ -67,7 +67,7 @@
 	void clickHangup( void );
 	void setHide( void );
 	bool isHided( void ) const { return hided; }
-	void KCallWidget::setDTMFSender( QString &s );
+	void setDTMFSender( QString &s );
 
 public slots:
 	void pleaseDial( const SipUri &dialuri );
diff -Nur kphone-orig/kphone/kphoneview.h kphone/kphone/kphoneview.h
--- kphone-orig/kphone/kphoneview.h	2005-04-18 13:04:40.000000000 +0000
+++ kphone/kphone/kphoneview.h	2006-04-03 11:59:52.206365500 +0000
@@ -54,7 +54,7 @@
 	QString getStunSrv( void );
 	void kphoneQuit( void );
 	bool getState( void );
-	KCallWidget *KPhoneView::DoCall( QString num, SipCall::CallType ctype );
+	KCallWidget *DoCall( QString num, SipCall::CallType ctype );
 
 signals:
 	void stateChanged( void );
